syft/serde.py

```diff
@@ -1332,17 +1354,18 @@ def _force_full_simplify(obj: object) -> object:
     MultiPointerTensor: [14, _simplify_multi_pointer_tensor],
     Plan: [15, _simplify_plan],
     VirtualWorker: [16, _simplify_worker],
-    GetNotPermittedError: [17, _simplify_GetNotPermittedError],
-    str: [18, _simplify_str],
-    pointers.ObjectWrapper: [20, _simplify_object_wrapper],
+    str: [17, _simplify_str],
```

Comment:
why is str no longer 18

Reply:
Because I removed 17:GetNotPermittedError as my exception ser/de supports also kinds of errors, so all numbers have been decremented after 17